packages/playground-ui/src/index.css

```
@@ -324,13 +325,19 @@ html.light {
-    'Noto Color Emoji';
+  /*
+   * Role-based font tokens. Components reference --font-display (headlines,
+   * brand) and --font-body (UI, paragraphs). Defaults are system fonts so
+   * this package has no font-licensing footprint. Consumers override these
+   * tokens in their own CSS to apply branded fonts (e.g., Greed, Geist).
+   */
+  --font-display: ui-serif, Georgia, Cambria, 'Times New Roman', Times, serif;
+  --font-body:
+    system-ui, ui-sans-serif, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
+  --font-mono: ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, 'Liberation Mono', 'Courier New', monospace;
+  /* Backward-compat aliases — Tailwind utilities `font-sans` / `font-serif` keep working. */
```

Stylelint (17.10.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)